src/middleware/assets.js

```diff
@@ -178,6 +178,12 @@ Or \`rm -rf bower_components/n-ui && bower install n-ui\` if you're no longer wo
 
 				res.locals.cssBundles.forEach(file => res.linkResource(file.path, {as: 'style'}));
```

Comment:
As discussed in person we need to allow a user to have both `preload` and `precache`